server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -398,6 +405,8 @@ private CommandExecutor startCommandExecutor(
                     return null;
                 });
             }
+            storagePurgeService.stop();
+            logger.info("Stopped the storage purge service on the leader replica ..");
```

Comment:
`..` means something is in progress. In this case, you finished starting the purge service already, so there's no need to append `..`.